app/models/ingredient_measure.rb

```diff
@@ -1,5 +1,9 @@
 class IngredientMeasure < ApplicationRecord
   belongs_to :ingredient
+
+  validates :name, presence: true, uniqueness: { scope: :ingredient_id }
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Rails/UniqueValidationWithoutIndex: Uniqueness validation should be with a unique index.